### PR TITLE
Empty Stats: AB test the nudges order

### PR DESCRIPTION
**Pull request by leandroalonso**
Only merge after #17313

### To test

1. Do a fresh install
2. Go to Stats
3. Notice the order in which the first card appears to you
4. Check in the nudge events that a property called `first_nudge` is fired with the correct value

## Regression Notes
1. Potential unintended areas of impact
-

2. What I did to test those areas of impact (or what existing automated tests I relied on)
-

3. What automated tests I added (or what prevented me from doing so)
-

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by momo-ozawa — APPROVED** [review]
Works as described!

`social` appears first. Do you think we should call it `publicize` instead?

```
2021-10-15 17:30:42:672 WordPress[17788:664620] 🔵 Tracked: stats_publicize_nudge_shown <blog_id: 198007634, first_nudge: social, site_type: blog>
2021-10-15 17:30:57:995 WordPress[17788:664620] 🔵 Tracked: stats_publicize_nudge_tapped <blog_id: 198007634, first_nudge: social, site_type: blog>
2021-10-15 17:31:05:610 WordPress[17788:664620] 🔵 Tracked: stats_publicize_nudge_dismissed <blog_id: 198007634, first_nudge: social, site_type: blog>
```

**Comment by leandroalonso** [comment]
> social appears first. Do you think we should call it publicize instead?

Definitely! I'll change it.

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -9,7 +9,6 @@
 * [**] Block editor: Embed block: Detect when an embeddable URL is pasted into an empty paragraph. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4048]
 * [**] Block editor: Pullquote block - Added support for text and background color customization [https://github.com/WordPress/gutenberg/pull/34451]
 * [**] Block editor: Preformatted block - Added support for text and background color customization [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4071]
-* [**] Stats: added Publicize and Blogging Reminders nudges for sites with low traffic in order to increase it.
```
Comment (momo-ozawa) [inline RELEASE-NOTES.txt:12]: Thanks! 🙏

File: WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift [hunk 2]
```diff
@@ -652,9 +652,11 @@ extension SiteStatsInsightsTableViewController: NoResultsViewHost {
 private extension SiteStatsInsightsTableViewController {
 
     func trackNudgeEvent(_ event: WPAnalyticsEvent) {
+        let firstNudge: String = EmptyStatsAB.shared.variant == .control ? "publicize" : "bloggingReminder"
```
Comment (momo-ozawa) [inline WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift:655]: 👍

**Review by momo-ozawa — APPROVED** [review]
🚀

**Pull request merged**